paymentsheet/src/main/java/com/stripe/android/customersheet/StripeCustomerAdapter.kt

```diff
@@ -200,10 +203,6 @@ internal class StripeCustomerAdapter @Inject internal constructor(
         return cacheDate + CACHED_CUSTOMER_MAX_AGE_MILLIS < nowInMillis
     }
 
-    private suspend fun isGooglePayAvailable(): Boolean {
-        return CustomerSheetHacks.configuration.await().googlePayEnabled
-    }
```

Comment:
Not needing to do this is nice. This effectively makes `StripeCustomerAdapter` a usable class now outside of `CustomerAdapter`.

Reply:
This was also a big blocker for process death handling. While `CustomerSheetActivity` persisted `Configuration` through its activity arguments, `StripeCustomerAdapter` requiring it means it always needed to be initialized after process death.